_data/meetups.yml

```diff
@@ -0,0 +1,26 @@
+- title: "Kickoff P2P and Decentralized Web Meetup"
+  location: Mozilla Community Space, Suite 500, 366 Adelaide Street West
+  locationLink: https://osm.org/go/ZX6BjZ50h?m=
+  meetup: meetup.com/p2p-and-dweb-toronto
```

Comment:
It doesn't look this prop is being used—for future use?

Reply:
I tried out having it as an `event-meta` element but it wasn't great :/ could remove or leave in :woman_shrugging: